std/StringTools.hx

```diff
@@ -517,25 +504,16 @@ class StringTools {
 		The input will be quoted, or escaped if necessary.
 	**/
 	@:noCompletion
+	@:deprecated('StringTools.quoteUnixArg() is deprecated. Use haxe.SysTools.quoteUnixArg() instead.')
 	public static function quoteUnixArg(argument:String):String {
-		// Based on cpython's shlex.quote().
-		// https://hg.python.org/cpython/file/a3f076d4f54f/Lib/shlex.py#l278
-
-		if (argument == "")
-			return "''";
-
-		if (!~/[^a-zA-Z0-9_@%+=:,.\/-]/.match(argument))
-			return argument;
-
-		// use single quotes, and put single quotes into double quotes
-		// the string $'b is then quoted as '$'"'"'b'
-		return "'" + replace(argument, "'", "'\"'\"'") + "'";
+		return inline haxe.SysTools.quoteUnixArg(argument);
 	}
 
 	/**
 		Character codes of the characters that will be escaped by `quoteWinArg(_, true)`.
 	**/
 	@:noCompletion
+	@:deprecated('StringTools.winMetaCharacters is deprecated. Use haxe.SysTools.winMetaCharacters instead.')
 	public static var winMetaCharacters = [" ".code, "(".code, ")".code, "%".code, "!".code, "^".code, "\"".code, "<".code, ">".code, "&".code, "|".code, "\n".code, "\r".code, ",".code, ";".code];
```

Comment:
Shouldn't this just be a reference to the new array?

Reply:
The new field has different type and access.

Reply:
Only at compile time though.. `cast`? :)

Reply:
Or it could be turned into a `get, never` property that redirects.

Reply:
Changed to `cast SysTools.winMetaCharacters`